packages/terra-graphs-docs/src/example-datasets/dataObjects/Line/dataset5noShapes.js

```diff
@@ -0,0 +1,23 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import { loadPopup } from '@cerner/terra-graphs-docs/lib/example-datasets/popup';
+import { createGraphValuesWithCos } from './createGraphValues';
+
+const data = {
+  key: 'uid_6',
+  label: {
+    display: 'Data Label 6',
```

Comment:
Should the key and display be `uid_5` and `Data Label 5` respectively to be consistent with the file name dataset5noShapes since that is how the other examples above are named?